docs/installation/slack/cloud-slack.mdx

```diff
@@ -74,6 +74,22 @@ After this point, all of your commands will be executed on the default instance.
 
 ![Cloud Slack All Clusters](assets/cloud_slack_command_all_clusters.png "Cloud Slack all clusters")
 
+### Setting Public Alias for Private Channels
+
+In order to safeguard your confidentiality while enabling seamless integration with Botkube plugins, it's essential to establish a public alias for your private Slack channels. This alias will be accessible solely to Botkube Cloud administrators.
```

Comment:
Replace this with: In order to maintain your confidentiality while using Botkube's plugins, you need to create a public alias for your private Slack channels. This alias will only be visible to Botkube Cloud administrators.

Reply:
resolved in https://github.com/kubeshop/botkube-docs/pull/321